src/commands/execute.rs

```diff
@@ -117,6 +117,9 @@ pub async fn common(cli: &mut Connection, cmd: &Common, options: &Options)
             MigrationCmd::Extract(params) => {
                 migrations::extract(cli, &options, params).await?;
             }
+            MigrationCmd::FormatUpgrade(params) => {
```

Comment:
Let's name this `UpgradeFormat` (`upgrade-format`) for consistency with `edgedb <group> <verb>`